packages/modules/init-capital/src/actions/supportNewPools.ts

```diff
@@ -0,0 +1,386 @@
+import { SetMaxHealthAfterLiqSubAction } from './subactions/setMaxHealthAfterLiq'
+import { SetModeAndTokenLiqMultiplierSubAction } from './subactions/setModeAndTokenLiqMultiplier'
+import { SetModeDebtCeilingInfosSubAction } from './subactions/setModeDebtCeilingInfos'
+import { SetModePoolFactorsSubAction } from './subactions/setModePoolFactors'
+import { SetModeStatusesDefaultSubAction } from './subactions/setModeStatusesDefault'
+import {
+  Api3Params,
+  LsdApi3Params,
+  PythParams,
+  SetNewPoolOracleReaderSubAction,
+  SetNewPoolOracleReaderSubActionParams,
+} from './subactions/setNewPoolOracleReader'
+import { SetPoolConfigSubAction } from './subactions/setPoolConfig'
+import { InitCapitalRegistry } from '@/src/type'
+
+export type ModeConfig = {
+  mode: number
+  poolConfig: {
+    collFactorE18: bigint
+    borrFactorE18: bigint
+    debtCeiling: bigint
+  }
+  config?: {
+    liqIncentiveMultiplierE18: bigint
+    minLiqIncentiveMultiplierE18: bigint
+    maxHealthAfterLiqE18: bigint
+  }
+}
+
+export const oracleReader = z.discriminatedUnion('type', [
+  z.object({ type: z.literal('api3'), params: z.custom<Api3Params>() }),
+  z.object({ type: z.literal('lsdApi3'), params: z.custom<LsdApi3Params>() }),
+  z.object({ type: z.literal('pyth'), params: z.custom<PythParams>() }),
+])
+
+export const oracleReaderRegistryName: Record<'api3' | 'lsdApi3' | 'pyth', keyof InitCapitalRegistry> = {
+  api3: 'api3ProxyOracleReaderProxy',
+  lsdApi3: 'lsdApi3ProxyOracleReaderProxy',
+  pyth: 'pythOracleReaderProxy',
+}
+
+export const SupportNewPoolActionParamsSchema = z.object({
+  name: z.string().describe(`Name of the pool e.g. POOL_WBTC`),
+  token: zodAddressNonZero.describe(`Address of the token e.g. WBTC`),
+  modeConfigs: z
+    .array(
+      z.object({
+        mode: z.number().describe(`Mode number starting from 0`),
+        poolConfig: z
+          .object({
+            collFactorE18: z.bigint().describe(`Collateral factor in E18, should be less than 1e18, e.g. parseUnit('0.8', 18)`),
+            borrFactorE18: z.bigint().describe(`Borrow factor in E18, should be greater than 1e18, e.g. parseUnit('1.1', 18)`),
+            debtCeiling: z
+              .bigint()
+              .describe(
+                `Debt ceiling for the pool in this mode, user could no longer borrow from the pool for this mode if the debt ceiling is reached`,
+              ),
+          })
+          .describe(`Pool config`),
+        config: z.object({
+          liqIncentiveMultiplierE18: z.bigint().describe(`Liq incentive multiplier e18`),
+          minLiqIncentiveMultiplierE18: z.bigint().describe(`Min liq incentive multiplier e18`),
+          maxHealthAfterLiqE18: z.bigint().describe(`Max Health after liq e18, if set to max uint64`),
+        }),
+      }) satisfies z.ZodType<ModeConfig>,
+    )
+    .describe(`mode configs for adding new mode`),
+  liqIncentiveMultiplierE18: z.bigint().describe(`liq incentive multiplier e18`).optional(),
```

Comment:
what is the diff between this line and the 75th line

Reply:
added description for being liqudationMultiplier for the Mode and Token